src/resource_clients/key_value_store.ts

```diff
@@ -18,6 +18,7 @@ import {
     parseDateFields,
     pluckData,
 } from '../utils';
+import { STORAGE_GENERAL_ACCESS } from '@apify/consts';
```

Comment:
this should be higher, before the relative imports. 

i'll take a look at the eslint config, it should have caught this, i think i know why its not working